### DISCUSSION TITLE
PSA: Advice for getting boosted on waitlist

### DISCUSSION THREAD

**Discussion by mccoydj1** [acheong08/EdgeGPT#70] [Announcements]
This is not an issue… more of a public service announcement on how to get access quickly and not be on the waitlist

Steps to get off waitlist (this worked for me)
- Log on to bing.com
- In the top right there is a number (it’s your achievement score in bing). Get that number above 100 and you get access from waitlist
- To increase number you can play bing games, do searches from bing, etc

**Comment by acheong08** [comment]
Thanks. Mine is at 57 and just got access

**Comment by MeowTechOpenSource** [comment]
... Mine is at 361 but still dont get access

**Comment by mccoydj1** [comment]
> ... Mine is at 361 but still dont get access

It might have something to do with your activity while on waitlist. Maybe try to raise it another 50 pts?

**Comment by xcloudx12345** [comment]
I'm already over 700 point when I'm in the waitlist, and I just got access today.

**Comment by Manamama** [comment]
Pure gamification, upselling, etc. sigh. Well, they had invested $$$ so it is their right to recoup it now and all 2023, obviously. 

Anybody could script fake installs and mouse clicks on a virtual Droid: Anbox plus MS boxes ;)? 
FYI, I have (re)installed their Bing smth crap on a protected Droid today, updated an unused Windoze cobwebbed box, both under the same IRL non-throwaway MS dev account (thus risking [this](https://github.com/acheong08/ChatGPT/issues/770), careful), and have 76 somethings just for starters, but still nothing, sigh.

**Comment by Yishaqel** [comment]
I doubt it is true, I have been waiting since day 7 to have access and I have been gaining points every day and I am still on the waiting list.

**Comment by acheong08** [comment]
It's pure luck

**Comment by LuckyHead11** [comment]
I have 8115 :)

**Comment by myfingerhurt** [comment]
my 18 points got access too. but waited for 3 or 4 days.

**Comment by r3l1c7** [comment]
i have 380, been on the waitlist for 9 days.

it's by region, I'm in the southeast USA.

**Comment by LuckyHead11** [comment]
I signed up about a week ago... with 8115 points I should have it by now :) Sadly I don't

**Comment by Manamama** [comment]
A tip that may work or not. At the right corner of that Edge thingy (also available as a DEB on Linux, hence one can script it, etc. without integration with the OS itself, plus full ready VMs and such), you have the "Other personas" toggle:
![Screenshot from 2023-02-17 13-54-34](https://user-images.githubusercontent.com/78492008/219658678-e285bae9-3e09-449f-bd89-c67b7e687c13.png)


Use it, it may help, as they are fully separate. 
I have unearthed my cobwebbed MS accounts from mid 2000s and 1990s, including at least one institutional one, and one set up as a minor, so as to 'tokenize it up' a bit more, now and in future.

**Comment by Manamama** [comment]
FYI, out of these three ancient (2000s?) MS accounts, at least one is working by now, but too late, given the rate limit, sigh. 
Apart from its somewhat hypnotising ¨me the good bot, okeyy?" allure, see [a short FYI here](https://github.com/acheong08/EdgeGPT/issues/18#issuecomment-1435699108), its kool-aid vibes made me quite like this Linux Bing by now. 
Should I seek professional (robotic?) help?

**Comment by LuckyHead11** [comment]
I had 8,295 and just got access, it took me 11 days but felt like 11 years :)
